src/lib/menu/menu-info.ts

```diff
@@ -135,5 +135,6 @@ export const MENU_INFO_MAP: Record<MenuId, MenuInfo> = Object.freeze({
     [MENU_ID.INFO_NOTICE]: {
         translationId: 'MENU.INFO_NOTICE',
         isNew: true,
+        needPermissionByRole: true,
```

Comment:
issue: If give true to `needPermissionByRole`, this menu will be exposed at [Role > Page Permission].

Reply:
Oh thanks I got it!